File: _정답코드/1주차/01_두_수의_합/solution.java
class Solution {
    public int solution(int[] numbers, int m) {
        int res = 0;
        int n = numbers.length;
        for (int i = 0; i < n - 1; i++) {
            for (int j = i + 1; j < n; j++) {
                int sum = numbers[i] + numbers[j];
                if (sum <= m && sum > res) {
                    res = sum;
                }
            }
        }
        return res;
    }
}

File: _정답코드/1주차/01_두_수의_합/solution.py
def solution(numbers, m):
    res = 0
    n = len(numbers)
    for i in range(n - 1):
        for j in range(i + 1, n):
            s = numbers[i] + numbers[j]
            if s <= m and s > res:
                res = s
    return res
